### ISSUE TITLE
PDFMergerUtility Producing Empty PDF

### ISSUE THREAD

**Issue by ddaeschler** [TomRoush/PdfBox-Android#133]
Good afternoon,

When using the PDF Merger on a newly created PDF, appending files is not working as expected and ends up producing a PDF with no pages.

```
for (int i = 0; i < files.size(); i++) {
            //load each file and append
            FileInputStream loadedFile = new FileInputStream(files.getString(i));
            PDDocument newDoc = PDDocument.load(loadedFile, true);

            PDFMergerUtility mu = new PDFMergerUtility();
            mu.appendDocument(this.document, newDoc);
            this.document.save(this.path);

            newDoc.close();
            loadedFile.close();
        }
```

The resulting PDF is:
```
%PDF-1.4
%צה
1 0 obj
<<
/Type /Catalog
/Version /1.4
/Pages 2 0 R
>>
endobj
2 0 obj
<<
/Type /Pages
/Kids []
/Count 0
>>
endobj
xref
0 3
0000000000 65535 f
0000000015 00000 n
0000000078 00000 n
trailer
<<
/Root 1 0 R
/ID [<F3FE6FBA5262A74273D126D5E1913D45> <F3FE6FBA5262A74273D126D5E1913D45>]
/Size 3
>>
startxref
130
%%EOF
```

The source PDFs contained scanned images.

**Comment by ddaeschler** [comment]
I just confirmed I can complete a merge of PDFs containing images with the upstream Apache PDFBox.

**Comment by TomRoush** [comment]
Possibly the same issue as #130 . Does the PDF you're trying to merge have any images in it?

**Comment by TomRoush** [comment]
The issue has been fixed in the develop-2.0 branch and the fix will be available with the 2.0 release.

**Comment by TomRoush** [comment]
Closing as 2.0.0.0 is now released
